pdf/model/colorspace.go

```diff
@@ -509,14 +509,17 @@ func (this *PdfColorspaceDeviceCMYK) ColorToRGB(color PdfColor) (PdfColor, error
 	y := cmyk.Y()
 	k := cmyk.K()
 
-	c = c*(1-k) + k
-	m = m*(1-k) + k
-	y = y*(1-k) + k
-
+	// Peter Williams. I need a cmyk->rgb conversion that gives r!=g or g!=b when c!=m or m!=y
```

Comment:
Thanks for those valuable explanations. Lots of useful insights, I remember finding some of the color specifications in the PDF standard rather vague, especially when it came to device vs device-independent.  Obviously PDFs are shared all over the place and are hardly bound to a device.

Do you think it would be useful to allow overriding some of those default conversion functions?  Could be nice if can be done in a kind of generic way.